### PR TITLE
Diff and code review refactors and improvements

### PR DESCRIPTION
**Pull request by silverwind**
- Simplify Diff CSS styling and add color variables for diff.
- Fix vertical centering of inline comment "show resolved" button.
- Slightly adjust text colors, e.g. in comment header.
- More CSS vars refactors.
- Reduce paddings and fix border on arc-green on inline review box.
- Fix color inconsistency on grey timeline badges.
- Very slightly brighten timeline in arc-green.

Screen Shot 2020-12-16 at 00 20 23
Screen Shot 2020-12-16 at 00 20 31
Screen Shot 2020-12-16 at 00 20 52
Screen Shot 2020-12-16 at 00 21 00

Screen Shot 2020-12-09 at 23 17 29
Screen Shot 2020-12-09 at 23 13 16

Screen Shot 2020-12-15 at 00 53 34
Screen Shot 2020-12-15 at 00 53 13
Screen Shot 2020-12-15 at 00 47 08
Screen Shot 2020-12-15 at 00 46 55

### CODE REVIEW

File: web_src/less/_repository.less [hunk 1]
```diff
@@ -1100,45 +1100,6 @@
...
-          &.octicon-comment {
-            font-size: 21px;
-            line-height: 33px;
-
-            &::before {
-              width: 20px;
-            }
-          }
-        }
-
```
Comment (silverwind) [inline web_src/less/_repository.less:1105]: I think this block is dead code that had been forgotten to be removed. At least I could not find a reference to this `issue-symbol` class.

**Comment by silverwind** [comment]
Included a few more fixes and improvments for code review:

- More CSS vars refactors.
- Reduce paddings and fix border on arc-green on inline review box.

**Comment by silverwind** [comment]
There's a regression on split diff left side background color, putton on draft til fix.

image

**Comment by silverwind** [comment]
Ok split diff should be sorted out. I Also happened to fix this split diff bug seen on master on gitea theme only:

https://try.gitea.io/silverwind/remap-css/commit/19283402eebd928cc8cde8ec1764520a764c648b?style=split

Screen Shot 2020-12-16 at 00 14 23

**Review by lunny — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work.

**Pull request merged**